extensions/phylum.ts

```diff
@@ -88,13 +88,15 @@ export class PhylumApi {
    * @param packages - List of packages to analyze
    * @param project - Project name. If undefined, the `.phylum_project` file will be used
    * @param group - Group name
+   * @param label - Analysis label for this job
    *
    * @returns Analyze Job ID, which can later be queried with `getJobStatus`.
    */
   static analyze(
     packages: Package[],
     project?: string,
     group?: string,
+    label?: string,
```

Comment:
This needs to also be added to the `DenoCore.opAsync()` call below.

Reply:
Already is.